### ISSUE TITLE
CA1062 false positive

### ISSUE THREAD

**Issue by tripplet** [dotnet/roslyn-analyzers#3864]
#### Analyzer package

[Microsoft.CodeAnalysis.FxCopAnalyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers)

#### Package Version

v3.0.0 (Latest)

#### Diagnostic ID

[CA1062](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1062)

#### Repro steps

In the minimal example below the analyzer detects a the CA1062 warning at the `action1(0);` which is a false positive.

```C#
    public class Class1
    {
        public void Test1(Action<int> action1)
        {
            _ = action1 ?? throw new ArgumentNullException(nameof(action1));

            this.Dummy();
            action1(0);
        }

        private void Dummy()
        {
            Action<string> a = null;
            a.Invoke("");
        }
    }
```

This is a very strange case because if the function body of dummy() is empty or only contains `Console.WriteLine("test");` no CA1062 is reported.

#### Expected behavior
No CA1062

#### Actual behavior
CA1062 at action1(0);

**Comment by mavasani** [comment]
This is fixed in latest prerelease 3.3.0-beta2. Can you please try?

**Comment by tripplet** [comment]
Yes it is fixed, thanks
